scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,151 @@
+  device_type: 'EQFP' #footprint names will be of style ...EQFP-pincount_...
+```
+
+---
+
+Every further entry in the script is assumed to be a package size definition. The top level parameter will give the internal parameter set name. This one must be unique in this file and should be representative of the footprint as it will be used in error messages. (It also makes later maintenance easier if it is easy to determine which footprint was generated by which parameter set.)
+
+``` yaml
+internal_package_name:
+  # parameter list (See below)
+```
+
+### Documentation and naming parameters
+- Size source to be added to footprint documentation field (`size_source`) {url}
+- Footprint name generation control
+  - Manufacturer name. Will be added as a prefix if given. (`manufacturer`) {string}
+  - Part number. Will be added as a prefix if given. (`part_number`) {string}
+  - Suffix: A custom suffix (added after pin pitch in default naming format). Can include parameters `pad_x` and `pad_y`.
+  - Custom naming (`custom_name_format`) {python format string}
+    - The full default format string is '{man:s}_{mpn:s}_{pkg:s}-{pincount:d}-1EP_{size_x:g}x{size_y:g}mm_P{pitch:g}mm{suffix:s}_EP{ep_size_x:g}x{ep_size_y:g}mm_Mask{mask_size_x:g}x{mask_size_y:g}mm{suffix2:s}{vias:s}' (The same parameters can be used in your custom format. Exposed pad parameters are not available for components without exposed pad.)
+
+_Note: Contributions intended for the official library shall not include the manufacturer or part number unless the footprint is specific to that manufacturer or part. Similarly avoid custom naming for official library contributions unless required to achieve the requested name (Example TI-specific naming)._
+
+### Package dimensions
+![dimension example](./documentation/dimension_system.svg)
+- Body size (`body_size_x`, `body_size_y`, `overall_height`) {dimension}
+- Lead dimensions:
+  - Overall size representing lead tip to lead tip dimension (`overall_size_x`, `overall_size_y`) {dimension}
+  - Lead width (`lead_width`) {dimension}
+  - Lead length representing tip till bend (`lead_len`)
+- lead pitch, currently equal for all sides (`pitch`) {float}
+- pin count (`num_pins_x`, `num_pins_y`) {int}
+  - num_pins_x=0 is used for generating SOIC like packages.
+  - num_pins_y=0 is used to generate SOIC like package footprints but with inverted pin numbering (Mirrored numbering scheme. Some manufactures use this style in their datasheets. Make sure you are not looking at the bottom view before using this. Not supported for QFP and similar.)
+
+### Exposed pad handling:
+![exposed pad example](../documentation/ep_handling.svg)
+- Size of package exposed pad or slug (`EP_size_x`, `EP_size_y`) {dimension}
+- Optional the size of the footprint pad (`EP_size_x_overwrite`, `EP_size_y_overwrite`) {float}
+   - Pad size is equal to nominal package pad size if not given.
+   - Use this to create a soldermask defined pad.
```

Comment:
'soldermask-defined' should have a hyphen.

Reply:
fixed

Reply:
This looks good, but also make the same change on line 74.